apps/forums/models.py

```diff
@@ -85,6 +85,13 @@ def recent_message(self):
 class ForumMessage(models.Model):
     thread = models.ForeignKey(ForumThread, related_name='messages')
     author = models.ForeignKey(settings.AUTH_USER_MODEL, related_name='messages')
+    reply_to = models.ForeignKey(
+        'self',
+        related_name='replies',
+        blank='True',
+        null='True',
```

Comment:
```suggestion
        blank=True,
        null=True,
```

Use the actual boolean value, not strings.